docs/types.rst

```diff
@@ -307,8 +313,9 @@ Members:
 * ``.length`` yields the fixed length of the byte array (read-only).
 
 .. note::
-    It is possible to use an array of bytes as ``byte[]``, but it is wasting a lot of space, 31 bytes every element,
-    to be exact, when passing in calls. It is better to use ``bytes``.
+    The type ``byte[]`` is an array of bytes, but due to padding rules, it will waste
```

Comment:
@chriseth Again, minor, but it's good to remove future tense in technical writing, so in this case, just "it wastes".